live/core/README.md

```diff
@@ -102,16 +102,44 @@ vpc_id=$(aws ssm get-parameter --name "$vpc_id_parameter_name" --query 'Paramete
 echo "VPC ID: $vpc_id"
 ```
 
-### Connecting to the Bastion Host
+## Connecting to the Bastion Host
 
-To establish a connection with the bastion host, use the following commands:
+To establish a secure connection with the bastion host, follow these steps:
+
+### 1. Obtain Required Information
+
+First, you need to gather some essential information:
+
+- Bastion SSH Parameter Name
+- SSH Key File Name
+- Bastion Instance ID
+
+You can retrieve these values using Terraform:
 
 ```bash
-# Retrieve the SSH command from the Terraform output
-ssh_command=$(terraform output -json | jq -r '.bastion_ssh_command.value')
+bastion_ssh_parameter_name=$(terraform output -json | jq -r '.ssm_parameter_bastion_ssh_key.value')
+ssh_key_file_name=$(terraform output -json | jq -r '.bastion_ssh_key_name.value') 
```

Comment:
we don't need to save the name as an output. we can name the file with any name 😊 just hardcode it in the README 👌🏻